src/util/config_macros.h

```diff
@@ -49,3 +48,9 @@
     }                                          \
     RETURN_STATUS(acrc);                       \
   }
+
+#ifdef RS_COORDINATOR
```

Comment:
It’s a subject to discuss. After dealing with config/debug commands, there is still some overhead that people might want to avoid in the standalone case